docs/technical-design/database-diagram.md

```diff
@@ -0,0 +1,110 @@
+# Diagram of the db with future designs
+
+## Current DB
+
+```mermaid
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+```
+
+## QA Doc Upload Design
+
+```mermaid
+
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+
+QuestionRound {
+    string id
+    string pkgID
+    string name
+    string requestedBy  
+    date requestDate
+    date responseDate
+}
+
+QuestionRound ||--|| User: "requestedBy"            
+
+QuestionRound ||--|{ Document: "has many"
+HealthPlanPackageTable ||--|{ QuestionRound: "has many"
+
+Document {
+    string name
+    string S3URL
+    string uploader
```

Comment:
What is uploader?

Reply:
I changed it to uploadedBy, we want to capture who uploaded this document to display that in the list

Reply:
gotcha. It should be `user uploadedBy` I think right? We want to start associating with our UserTable.